### DISCUSSION TITLE
Neat maze

### DISCUSSION THREAD

**Discussion by withinboredom** [Ijee/Amazeing#6] [Show and tell]
Here's a maze 
[amazeing.txt](https://github.com/user-attachments/files/16337980/amazeing_a-star_2024-07-22.txt)

This one is pretty cool because it shows different paths depending on the algorithms/heuristics chosen.

**Comment by Ijee** [comment]
This is indeed a good one. Nice find👍

I also just noticed that when importing it doesn't set the import as a save point. Meaning when you do the soft reset it gets back to 
an empty grid. I'll fix that soon :)
